examples/http-resource-error-handling/http_resource_error_handling.md

```diff
@@ -0,0 +1,19 @@
+# HTTP service - Error handling
+
+Error handling is an integral part of any network program. The error can be returned or checked from the resource function, interceptor, request dispatcher, or data binder. These errors are often handled by a default handler and sent back as `500 InternalSeverError` responses with the error message in the body. The error is printed on the server terminal along with the stacktrace. This prevents some additional error-handling codes in your program.
+
+>**Note:** Use `on fail` to group few errors or to customize the `check` error as described in [Check semantics](/learn/by-example/check-semantics/) example. Alternatively, add a response error interceptor as described in [HTTP service - Interceptor error handling](/learn/by-example/http-interceptor-error-handling/) example to more flexibility.
```

Comment:
```suggestion
>**Note:** Use `on fail` to group several errors together or to customize the `check` error as described in [Check semantics](/learn/by-example/check-semantics/) example. Alternatively, add a response error interceptor as described in [HTTP service - Interceptor error handling](/learn/by-example/http-interceptor-error-handling/) example to more flexibility.
```

Reply:
Fixed with [383a899b20c1d742673453709a6143e038f0540d](https://github.com/ballerina-platform/ballerina-distribution/pull/3895/commits/383a899b20c1d742673453709a6143e038f0540d). Hence. marking the comment as resolved.